alfresco-apa-java-rest-api/alfresco-apa-java-rest-api-lib/generated/alfresco-activiti-admin-preference-rest-api/pom.xml

```diff
@@ -10,7 +10,6 @@
   </parent>
   <artifactId>alfresco-activiti-admin-preference-rest-api</artifactId>
   <name>alfresco-activiti-admin-preference-rest-api</name>
-  <version>7.14.0-SNAPSHOT</version>
```

Comment:
@matteo4diani the version can be removed from here because it's the same as the parent. In this case, it's better to don't set it and just inherit from the parent.